README.md

```diff
@@ -1,3 +1,273 @@
+
+    ```bash
+    mkdir landscape
+    cd landscape
+    git clone "https://github.com/gardener/garden-setup" crop
+    ```
+
+1. In your landscape folder, create a configuration file called `acre.yaml`. The structure of the configuration file is described [below](#configuration-file-acreyaml).
+
+    > Do not use file `acre.yaml` in folder `crop`. This file is used internally by the installation tool.
+
+1. If you created the base cluster using GKE you need to convert your `KUBECONFIG` to one that uses basic authentication with Google-specific configuration parameters:
+
+    1. Set your `KUBECONFIG` environment variable to a file that does not exist. Do *not* use the one specified in your acre.yaml file.
+
+    1. Download the GKE kubeconfig:
+
+        ```bash
+        gcloud container clusters get-credentials <your_cluster> --zone <your_zone> --project <your_project>
+        ```
+
+    1. Call `crop/bin/convertkubeconfig 3.1`. When asked for credentials, enter the ones that the GKE dashboard shows when clicking on `show credentials`.
+
+    This will create a kubeconfig that uses basic auth. It will be created at the path specified in your `acre.yaml` file.
+
+1. Open a second terminal window and enter the following command to watch the progress of the Gardener installation:
+
+    ```bash
+    watch -d kubectl -n garden get pods,ingress,sts,svc
+    ```
+
+1. In your first terminal window, use the following command to deploy Gardener on your base cluster:
+
+    ```bash
+    sow deploy -A
+    ```
+
+`sow` now starts to install Gardener in your base cluster. The installation can take about 30 minutes. `sow` prints out status messages to the terminal window so that you can check the state of the installation. The other terminal window will show the newly created Kubernetes resources after a while and if their deployment was successful (their status should be `Running`). Finally, the Gardener dashboard is deployed. The last status message before `sow` exits is something like this:
+```bash
+===================================================================
+Dashboard URL -> https://gardener.ingress.<value of `landscape.cluster.domain`>
+===================================================================
+generating exports
+*** species dashboard deployed
+```  
+
+More information: [Most Important Commands and Directories](#most-important-commands-and-directories)
+
+## Configuration File acre.yaml
+
+This file will be evaluated using `spiff`, a dynamic templating language for yaml files. For example, this simplifies the specification of field values that are used multiple times in the yaml file. For more information, see the [spiff repository](https://github.com/mandelsoft/spiff/blob/master/README.md).
+
+<pre>
+landscape:
+  <a href="#landscapename">name</a>: &lt;URL-friendly name&gt;                       # Gardener managed k8s landscape name
```

Comment:
I would use here something else as an example. Instead of "URL friendly" we should use something like "my-gardener".

Reply:
I am not convinced ;-)
My intention was to give readers a hint where this name will be used later.

What is more, `my-gardener` is an example, not a description of the field. If we use an example, we should omit the brackets (&lt; &gt;), but then it looks like a constant value.

Proposal:
```yaml
landscape:
  name: <Identifier>                  # general landscape identifier, for example, `my-gardener`
```